mappings/net/minecraft/block/entity/CampfireBlockEntity.mapping

```diff
@@ -12,3 +12,6 @@ CLASS net/minecraft/class_3924 net/minecraft/block/entity/CampfireBlockEntity
 	METHOD method_17507 saveInitialChunkData (Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487;
 		ARG 1 tag
 	METHOD method_17510 updateListeners ()V
+	METHOD method_31666 tickLit (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_3924;)V
+	METHOD method_31667 tickUnlit (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_3924;)V
```

Comment:
`litTick` and `unlitTick`?

Also is this on logical server or logical client?

Reply:
Logical server - there's another tick method for client only ticks. (Should I rename this to `litServerTick/unlitServerTick`?)

Reply:
Hmm yeah that seems good.